Neos.Workspace.Ui/Resources/Private/Fusion/Features/Restore/Components/NodesTableRow.fusion

```diff
@@ -0,0 +1,96 @@
+##
+# Renders a single nodes list item
+#
+prototype(Neos.Workspace.Ui:Component.NodesTableRow) < prototype(Neos.Fusion:Component) {
+  /// Neos\Workspace\Ui\ViewModel\RestoreListItem
+  restorelistItem = ${[]}
+  /// integer
+  level = 0
+
+  workspaceName = ''
+
+  @private {
+    i18n = ${I18n.id('').source('Main').package('Neos.Workspace.Ui')}
+
+    restoreUri = Neos.Fusion:ActionUri {
+      action = 'restoreNodeConfirmation'
+      format = 'htmx'
+      arguments {
+        workspaceName = ${props.workspaceName}
+        nodeAggregateId = ${props.restorelistItem.nodeAggregateId}
+      }
+    }
+    hardDeleteConfirmationUri = Neos.Fusion:ActionUri {
+      action = 'hardDeleteConfirmation'
+      format = 'htmx'
+      arguments {
+        nodeAggregateId = ${props.restorelistItem.nodeAggregateId}
+      }
+    }
+    hardDeleteConfirmationPopoverId = 'hard-delete-confirm-modal'
+
+    restoreConfirmationPopoverId = 'restore-confirm-modal'
+  }
+
+  renderer = afx`
+    <tr>
+      <td class="node-type-column">
+        <Neos.Workspace.Ui:Component.Icon icon={props.restorelistItem.icon}/>
+      </td>
+      <td>
+        <span>{props.restorelistItem.nodeTypeLabel}</span>
+      </td>
+      <td></td>
+      <td>
+        <span>{props.restorelistItem.deletionUserName}</span>
+      </td>
+      <td>
+        <span @if.hasDeletionDate={props.restorelistItem.deleteTime}>{Date.format(props.restorelistItem.deleteTime, 'Y-m-d H:i')}</span>
+        <span @if.hasNoDeletionDate={!props.restorelistItem.deleteTime}>-</span>
+      </td>
+      <td class="restore-action-column">
+        <Neos.Workspace.Ui:Component.Button
+          title={private.i18n.id('restore.actions.restoreChange')}
+          icon="fa-solid fa-trash-can-arrow-up"
+          label={private.i18n.id('restore.actions.restoreChange')}
+          attributes.hx-get={private.restoreUri}
+          attributes.hx-target='#popover-container'
+          attributes.hx-swap='innerHTML'
+          attributes.hx-on--after-request={'document.getElementById("' + private.restoreConfirmationPopoverId + '").showPopover()'}
+
+        />
+        <Neos.Workspace.Ui:Component.Button
+          isDanger
+          title={private.i18n.id('restore.actions.hardDelete')}
+          icon="trash-alt icon-white"
+          disabled={!Security.hasAccess('Neos.Workspace.Ui:Backend.HardDeleteNodes')}
```

Comment:
Hard delete should only be possible for nodes that exist and are soft deleted in the live workspace

Reply:
Hard delete button is now only shown for live workspace. Since the live workspace is not shown in the overview, currently the hard delete functionality is never shown